task/deprecated-image-check/0.5/deprecated-image-check.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   labels:
-    app.kubernetes.io/version: "0.3"
+    app.kubernetes.io/version: "0.5"
```

Comment:
I'm little confused here, the Migration MR says migration from 0.4 to 0.5 but this differs. Looks like it was forgotten to increment a version to 0.4. Jut to be aware of this fact.